docs/how-to-guides/run-and-evaluate-a-flow/index.md

```diff
@@ -241,4 +242,5 @@ Learn more about:
 
 use-column-mapping
 manage-runs
+compute-session
```

Comment:
This cloud only

Reply:
OK

Reply:
not like this, we shall have a cloud/run-and-evaluate-run